packages/syft/src/syft/service/worker/worker_image.py

```diff
@@ -91,28 +89,9 @@ class SyftWorkerImage(SyftObject):
     image_hash: Optional[str]
     created_at: DateTime = DateTime.now()
     created_by: SyftVerifyKey
+    built_on: Optional[DateTime]
 
-
-def build_using_docker(
```

Comment:
moved to `worker_image_service.py` function called `docker_push()`

Reply:
can we move this function to `service/worker/utils.py` instead ?